.drone.yml

```diff
@@ -2,7 +2,7 @@ image: go1.2
 script:
   - ./test_drone.sh
 services:
-  - clever/mongo:2.6
```

Comment:
probably should change this back to 2.6. unless the tests pass with this, in which case we can keep it at 2.4